### PR TITLE
fix: Update image in how-to testimonials page (#8814)

### LINKED ISSUE #8814: [BUG] Screenshots to change name from 'LinkFree' to 'BioDrop'
### Description

Edit: all screenshots need updating. 

In the https://www.biodrop.io/docs/how-to-guides/testimonials-json -> Giving a testimonial section ->
the name in the screenshot should be updated from 'LinkFree' to 'BioDrop'

### Screenshots

image

### Additional information

_No response_

### PR DESCRIPTION
**Pull request by rahulsiddarth**
Update BioDrop testimonial issue image in testimonials-json.mdx

## Fixes Issue

Closes #8814 

## Changes proposed

Update BioDrop testimonial issue image in how-to testimonials page with the new application name (LinkFree -> BioDrop)

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Screenshots

Screenshot 2023-08-30 170925

Screenshot 2023-08-30 170842

### CODE REVIEW

File: pages/docs/how-to-guides/testimonials-json.mdx [hunk 1]
```diff
@@ -14,7 +14,7 @@ You can give a Testimonial for another BioDrop user by going to their Profile an
 
 The issue will look like this:
 
-![BioDrop testimonial issue](https://user-images.githubusercontent.com/624760/210427249-25848dd6-463a-4748-aa60-45c6e53f49a8.png)
+![BioDrop testimonial issue]https://github.com/EddieHubCommunity/BioDrop/assets/43289009/15f7d441-1b68-424a-99d5-00fe8bc47537)
```
Comment (eddiejaoude) [inline pages/docs/how-to-guides/testimonials-json.mdx:17]: This is not valid markdown

**Review by eddiejaoude — CHANGES_REQUESTED** [review]
Thank you for your contribution. But the issue is to update all images used in the docs

File: pages/docs/how-to-guides/testimonials-json.mdx (hunk 1 above)
Comment (xkrishguptaa) [inline pages/docs/how-to-guides/testimonials-json.mdx:17]: Like Eddie mentioned, you just have to add this `(`

```suggestion
![BioDrop testimonial issue](https://github.com/EddieHubCommunity/BioDrop/assets/43289009/15f7d441-1b68-424a-99d5-00fe8bc47537)
```

**Comment by SaraJaoude** [comment]
> Hello Eddie, requesting that this issue be reassigned to someone who has a better screen size. Currently, with my 16:9 screen ratio , I'm unable to capture all the details like how it is present now in the screenshots.

@rahulsiddarth thank you for the work you did and I will unassigned the issue and also as a result close this PR.

**Pull request closed without merging**